conceptual/Npgsql/replication.md

```diff
@@ -65,6 +65,11 @@ await foreach (var message in conn.StartReplication(
     slot, new PgOutputReplicationOptions("blog_pub"), cancellationTokenSource.Token))
 {
     Console.WriteLine($"Received message type: {message.GetType().Name}");
+
+    // Always assign LastAppliedLsn and LastFlushedLsn so that Npgsq can inform the
```

Comment:
BTW if you set LastFlushedLsn, does setting LastAppliedLsn have any effect? Intuitively it seems like the former "implies" the latter - if that's really the case maybe we can remove it from here.

Reply:
I'm actually not absolutely sure if it's handled by Postgres as two levels of the same concept or as two independent concept. This is  - once again - something where I'll have at the backend source code to be sure.
For now the safest advice is probably to always assign both.

Reply:
OK